apps/core/.webpack/config.ts

```diff
@@ -121,6 +123,65 @@ export const createConfiguration: (
         '.mjs': ['.mjs', '.mts'],
       },
       extensions: ['.js', '.ts', '.tsx'],
+      modules: [
+        // Root directory's node_modules takes precedence over other node_modules.
+        resolve(rootPath, '../../', 'node_modules'),
+        'node_modules',
+      ],
+      fallback:
+        blocksuiteBaseDir === undefined
+          ? undefined
+          : {
+              events: require.resolve('events'),
```

Comment:
why we have this

Reply:
Module `events` is introduced in blocksuite, I have removed it by `event: false`